app/api/endpoints/plugins.py

```
@@ -0,0 +1,58 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from app.models.plugin import (
+    PluginConfigurationCreate,
+    PluginConfigurationUpdate,
+    PluginConfigurationOut,
+)
+from app.services.plugin_service import PluginService
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)